tests/python/python_models_test.py

```diff
@@ -0,0 +1,49 @@
+from rascal.representations import SphericalInvariants
+from rascal.models import Kernel
+from test_utils import load_json_frame, BoxList, Box
+import unittest
+import numpy as np
+import sys
+import json
+
+
+class TestCosineKernel(unittest.TestCase):
+    def setUp(self):
+        """
+        builds the test case. Test the order=1 structure manager implementation
+        against a triclinic crystal.
+        """
+
+        fn = 'reference_data/CaCrP2O7_mvc-11955_symmetrized.json'
+        self.frame = load_json_frame(fn)
+
+        self.hypers = dict(soap_type="PowerSpectrum",
+                           interaction_cutoff=3.5,
+                           max_radial=6,
+                           max_angular=6,
+                           gaussian_sigma_constant=0.4,
+                           gaussian_sigma_type="Constant",
+                           cutoff_smooth_width=0.5,
+                           )
+
+    def test_model_call(self):
+
+        rep = SphericalInvariants(**self.hypers)
+
+        features = rep.transform([self.frame])
+
+        for target_type in ["Atom", "Structure"]:
+            cosine_kernel = Kernel(rep, name="Cosine", target_type=target_type, zeta=2)
+            cosine_kernel(features)
+
+        # wrong name 
+        with self.assertRaises(RuntimeError):
+            Kernel(rep, name="WrongName", target_type="Structure", zeta=2)
+        # wrong target_type
+        with self.assertRaises(RuntimeError):
+            Kernel(rep, name="Cosine", target_type="WrongType", zeta=2)
```

Comment:
how about adding the wrong case (like "cosine" or "structure") to the list of assertRaises constructors?